packages/extension/src/components/icon/information-plain.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from "react";
+import { IconProps } from "./types";
+
+export const InformationPlainIcon: FunctionComponent<IconProps> = ({
```

Comment:
@renfengshi Could you look at another icon file and follow the convention for icon files?